### PR TITLE
Remove return stmt after yield from trigger

### PR DESCRIPTION
**Pull request by pankajastro**
Closes: #271 
Remove unused return statement from all trigger run methods i.e now trigger run method will only yield

### CODE REVIEW

File: astronomer/providers/amazon/aws/triggers/redshift_cluster.py [hunk 1]
```diff
@@ -58,20 +58,16 @@ async def run(self) -> AsyncIterator["TriggerEvent"]:  # type: ignore[override]
                 response = await hook.resume_cluster(cluster_identifier=self.cluster_identifier)
                 if response:
                     yield TriggerEvent(response)
-                    return
                 else:
                     error_message = f"{self.task_id} failed"
                     yield TriggerEvent({"status": "error", "message": str(error_message)})
```
Comment (pankajkoti) [inline astronomer/providers/amazon/aws/triggers/redshift_cluster.py:63]:
```suggestion
                    yield TriggerEvent({"status": "error", "message": error_message})
```
Not your change, but error_message on line 62 is already a string.

File: astronomer/providers/amazon/aws/triggers/redshift_cluster.py [hunk 2]
```diff
@@ -58,20 +58,16 @@ async def run(self) -> AsyncIterator["TriggerEvent"]:  # type: ignore[override]
...
-                    return
             else:
                 response = await hook.pause_cluster(cluster_identifier=self.cluster_identifier)
                 if response:
                     yield TriggerEvent(response)
-                    return
                 else:
                     error_message = f"{self.task_id} failed"
                     yield TriggerEvent({"status": "error", "message": str(error_message)})
```
Comment (pankajkoti) [inline astronomer/providers/amazon/aws/triggers/redshift_cluster.py:70]:
```suggestion
                    yield TriggerEvent({"status": "error", "message": error_message})
```

File: astronomer/providers/databricks/triggers/databricks.py [hunk 3]
```diff
@@ -63,11 +63,9 @@ async def run(self) -> AsyncIterator["TriggerEvent"]:  # type: ignore[override]
                             "run_page_url": self.run_page_url,
                         }
                     )
-                    return
                 else:
                     error_message = f"{self.task_id} failed with terminal state: {run_state}"
                     yield TriggerEvent({"status": "error", "message": str(error_message)})
```
Comment (pankajkoti) [inline astronomer/providers/databricks/triggers/databricks.py:68]:
```suggestion
                    yield TriggerEvent({"status": "error", "message": error_message})
```

File: astronomer/providers/snowflake/triggers/snowflake_trigger.py [hunk 4]
```diff
@@ -60,11 +60,8 @@ async def run(self) -> AsyncIterator["TriggerEvent"]:  # type: ignore[override]
             run_state = await hook.get_query_status(self.query_ids)
             if run_state:
                 yield TriggerEvent(run_state)
-                return
             else:
                 error_message = f"{self.task_id} failed with terminal state: {run_state}"
                 yield TriggerEvent({"status": "error", "message": str(error_message), "type": "ERROR"})
```
Comment (pankajkoti) [inline astronomer/providers/snowflake/triggers/snowflake_trigger.py:65]:
```suggestion
                yield TriggerEvent({"status": "error", "message": error_message, "type": "ERROR"})
```

**Review by pankajkoti — APPROVED** [review]

File: astronomer/providers/amazon/aws/triggers/emr.py [hunk 5]
```diff
@@ -61,14 +61,11 @@ async def run(self) -> AsyncIterator["TriggerEvent"]:  # type: ignore[override]
                 elif query_status in ("FAILED", "CANCELLED", "CANCEL_PENDING"):
                     msg = f"EMR Containers sensor failed {query_status}"
                     yield TriggerEvent({"status": "error", "message": msg})
-                    return
                 else:
                     msg = "EMR Containers sensors completed"
                     yield TriggerEvent({"status": "success", "message": msg})
-                    return
```
Comment (phanikumv) [inline astronomer/providers/amazon/aws/triggers/emr.py:68]: Can you please check that integration test dont break post this change

**Review by phanikumv — CHANGES_REQUESTED** [review]

**Review by bharanidharan14 — COMMENTED** [review]
Overall looks ok to me,Need to fix circleci failure

**Comment by pankajastro** [comment]
Look like the coverage percentage dropped because of deletion of `return` statement

File: astronomer/providers/amazon/aws/triggers/emr.py (hunk 5 above)
Comment (pankajastro) [reply astronomer/providers/amazon/aws/triggers/emr.py:68]: I tested for one dag `redshift_cluster_management` will run once all

**Review by bharanidharan14 — APPROVED** [review]

**Review by rajaths010494 — APPROVED** [review]

**Review by phanikumv — APPROVED** [review]

**Pull request merged**